src/ClaimAction.sol

```diff
@@ -179,7 +178,7 @@ contract ClaimAction is IClaimAction {
         uint256 gnoUsd = uint256(getChainlinkGnoUsdDataFeedLatestAnswer());
         uint256 eurUsd = uint256(getChainlinkEurUsdDataFeedLatestAnswer());
         require(eurUsd > 0, "EUR/USD feed is 0");
-        return (gnoUsd * 1e18) / eurUsd;
+        return (gnoUsd * 1e8) / eurUsd;
```

Comment:
`gnoUsd` is 8 decimals, `eurUsd` is 8 decimals, and both GNO and EURe tokens are 18 decimals.

That means `chainlinkGnoEurPrice()` returns 8 decimals values.

The require line 92 thus compares `eureAmount` the amount of EURe received from the swap (18 decimals) to `expectedEure = (amount * chainlinkPrice) / 1e18` which is `18 * 8 / 18 = 8` decimals. 

I think `gnoUsd * 1e18 / eurUsd` was correct.